kotlinx.fuzz.jazzer/src/main/kotlin/kotlinx/fuzz/jazzer/JazzerLauncher.kt

```diff
@@ -54,6 +72,30 @@ object JazzerLauncher {
         exitProcess(0)
     }
 
+    private fun Throwable.filter(): Throwable {
+        val filteredCause = cause?.filter()
+        val oldStackTrace = stackTrace
+
+        val newThrowable = (when {
+            message == null && filteredCause == null -> this::class.java.getConstructor().newInstance()
```

Comment:
Are we sure that this will not cause any problems? What if the exception class does not have some of these constructors (or they are private)?

Reply:
Fixed